src/makeRobot.js

```diff
@@ -38,7 +38,70 @@
  * @return {Robot}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const EVACUATE_X = 1400;
+  const EVACUATE_Y = 500;
+
+  return {
+    name,
+    wheels,
+    version,
+    get info() {
+      // I've found this soulution, but I don't like the code format.
+      // May you give me an advice?
+
+      return `name: ${this.name}, \
```

Comment:
```suggestion
      return `name: ${this.name}, chip version: ${this.version}, ` +
        `wheels: ${this.wheels}`;
```
It seems a bit better in terms of readability